localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -393,7 +418,7 @@ void NDTScanMatcher::callback_sensor_points(
       transformation_msg_array, oscillation_threshold_, inversion_vector_threshold_);
   }
   bool is_ok_converged_param = validate_converged_param(
-    ndt_result.transform_probability, ndt_result.nearest_voxel_transformation_likelihood);
+    no_ground_transform_probability, no_ground_nearest_voxel_transformation_likelihood);
```

Comment:
We should leave `transform_probability` and `nearest_voxel_transformation_likelihood` as it is right now, since other Autoware users (including TIER IV) are still using them. I believe its better to leave it as an option whether to use `no_ground_transform_probability` so that the users can choose on their own.